src/bashly.yml

```diff
@@ -42,6 +41,26 @@ commands:
         required: true
     examples:
       - dcu logo coreboot.rom -l bootsplash.bmp
+  - name: smmstore
+    help: Edit settings stored in SMMSTORE
+    args:
+      - name: dasharo_rom_file
+        required: true
+        help: Dasharo firmware file (e.g. coreboot.rom)
+    flags:
+      - long: --get
+        arg: var
+        help: Name of an SMMSTORE variable
+      - long: --list
+        help: List available variables
+      - long: --set
+        arg: var
+        help: Name of an SMMSTORE variable
+      - long: --value
+        arg: val
+        help: Value to set in a variable
+    examples:
+      - dcu smmstore coreboot.rom --get EnableWifiBt
```

Comment:
@SergiiDmytruk Since you have developed the tool, what happens if user would provide incorrect variable name (e.g. a typo), or incorrect type / value range? Do we need some validation here in terms of allowed list of variables, and allowed types and ranges for each of them implemented in this tool?

This DCU tool goes to end users and customers of Dasharo, so it must be more user friendly and preventing end users from making silly mistakes (like a typo or type mismatch) with no feedback from the tool.

Reply:
These types of scenarios should be covered by the tests: https://github.com/Dasharo/dcu?tab=readme-ov-file#testing

Reply:
It validates input and prints errors, although depending on the source of an error they might not necessarily be obvious to the end user:
* `-t boola`: `Failed to parse type: boola`
* `-n SmmBwpk`: `Couldn't find variable "dasharo:SmmBwpk"`
* `-t bool -v sss`: `Invalid boolean value: "sss"`, `Failed to parse value "sss" as bool`

Making a typo in variable's name on `set` won't be caught because there is no whitelist of variable names, so you can create/edit any variable you wish.  If needed, that would have to be implemented in DCU.

Reply:
I guess we can have a static mapping of variable names and their types in the script. It's not pretty, but it's foolproof and allows for neat handling of enums like ME mode. I got something like this:

```
./dcu smmstore current.rom --list                                                        smmstore!
Settings that can be modified using this tool:
NAME	VALUE	ACCEPTED VALUES
LockBios	Enabled	Enabled / Disabled
NetworkBoot	Disabled	Enabled / Disabled
UsbDriverStack	Enabled	Enabled / Disabled
UsbMassStorage	Enabled	Enabled / Disabled
SmmBwp	Disabled	Enabled / Disabled
Ps2Controller	Enabled	Enabled / Disabled
BootManagerEnabled	Enabled	Enabled / Disabled
PCIeResizeableBarsEnabled	Disabled	Enabled / Disabled
EnableCamera	Enabled	Enabled / Disabled
EnableWifiBt	Enabled	Enabled / Disabled
SerialRedirection	Disabled	Enabled / Disabled
SerialRedirection2	Disabled	Enabled / Disabled
```